pkg/roachprod/install/cluster_synced.go

```diff
@@ -1222,6 +1222,10 @@ func (c *SyncedCluster) runCmdOnSingleNode(
 		detailMsg := fmt.Sprintf("Node %d. Command with error:\n```\n%s\n```\n%s", node, cmd, output)
 		res.Err = errors.WithDetail(res.Err, detailMsg)
 	}
+	if res.RemoteExitStatus == 100 && (strings.Contains(nodeCmd, "apt") || strings.Contains(nodeCmd, "apt-get")) {
```

Comment:
Should we capture other variations, e.g., `apt install`? Are there other error codes specific to `apt` that we should maybe check here?

Reply:
Moved the logic to `ClassifyCmdError` and not checking for `apt` in the command anymore since it is _pretty unlikely_ that other commands also return a 100 exit code.